zerver/migrations/0324_backfill_scheduledemail_data_to_jsonfield.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.8 on 2021-04-30 17:02
+import orjson
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def backfill_edit_scheduledemail(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    ScheduledEmail = apps.get_model("zerver", "ScheduledEmail")
+
+    for scheduledemail in ScheduledEmail.objects.filter(data__isnull=False):
+        data_json = orjson.loads(scheduledemail.data)
+        scheduledemail.data_json = dict(data_json)
```

Comment:
Why this `dict()` line here?

Reply:
I wanted to copy the `data_json` dictionary to `data_json` field of the model, that is why I used `dict(data_json)` .. but now that you have mentioned, I realize that I should have done
`scheduleemail.data_json = data_json` ... will update it!